### PR TITLE
Add utility to fix ARP flux

### PR DESCRIPTION
**Pull request by turtle-fly**
Run with `sudo`

### CODE REVIEW

**Comment by turtle-fly** [comment]
@fub2 @xiar @XiaowenJiang @chenge3 @WinnieXU1 please review.

Unfortunately, this won't help on NIC teaming issue.

File: diag_arp_flux/diag_arp_flux.py [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+
+
+ROUTE_TABLE_PATH = "/etc/iproute2/rt_tables"
+TABLE_PREFIX = "infrasim"
+TABLES_TO_REMOVE = []
+TABLES_TO_DEFINE = []
+TABLE_USED_ID = []
+BASE_TABLE_ID = 200
+
+# dict_subnet can be in a form of
```
Comment (fub2) [inline diag_arp_flux/diag_arp_flux.py]: Nice structure definition!

File: diag_arp_flux/diag_arp_flux.py [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+                cmd_ip_route_on_nic = "ip route add {} via {} dev {} table {}".\
+                    format(subnet, info["ip"], info["if_name"], TABLE_PREFIX+info["if_name"])
+                ret, rsp = run_command(cmd_ip_route_on_nic)
+                if ret != 0:
+                    print "    Fail to run: {}".format(cmd_ip_route_on_nic)
+                    exit(1)
+                print "    {}".format(cmd_ip_route_on_nic)
+
+                gateway = default_gateway(subnet)
+                cmd_ip_route_default = "ip route add default via {} dev {} table {}".\
```
Comment (fub2) [inline diag_arp_flux/diag_arp_flux.py]: [minor]run one time is enough for one interface. it could be move to upper layer iteration.

**Comment by fub2** [comment]
Really nice work, very comprehensive! Ship it!!

**Pull request merged**